data/translations/kdiskmark_cs_CZ.ts

```diff
@@ -0,0 +1,441 @@
+        <source>Confirmation</source>
+        <translation>Potvrzení</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="452"/>
+        <source>This action destroys the data in %1
+Do you want to continue?</source>
+        <translation>Touto akci budou přepsána data v %1.
+Chcete pokračovat?</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="465"/>
+        <source>Benchmark Failed</source>
+        <translation>Test se nezdařil</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="399"/>
+        <location filename="../../src/mainwindow.cpp" line="400"/>
+        <location filename="../../src/mainwindow.cpp" line="401"/>
+        <location filename="../../src/mainwindow.cpp" line="402"/>
+        <location filename="../../src/mainwindow.cpp" line="403"/>
+        <source>Stop</source>
+        <translation>Stop</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../../src/main.cpp" line="53"/>
+        <source>No FIO was found. Please install FIO before using KDiskMark.</source>
+        <translation>FIO nenalezeno. Aby bylo možné používat KDiskMark, je třeba nainstalovat FIO.</translation>
+    </message>
+    <message>
+        <location filename="../../src/global.cpp" line="13"/>
+        <source>&lt;h1&gt;%1 MB/s&lt;br/&gt;%2 GB/s&lt;br/&gt;%3 IOPS&lt;br/&gt;%4 μs&lt;/h1&gt;</source>
+        <translation>&lt;h1&gt;%1 MB/s&lt;br/&gt;%2 GB/s&lt;br/&gt;%3 IOPS&lt;br/&gt;%4 μs&lt;/h1&gt;</translation>
+    </message>
+</context>
+<context>
+    <name>Settings</name>
+    <message>
+        <location filename="../../src/settings.ui" line="14"/>
+        <source>Settings</source>
+        <translation>Nastavení</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="29"/>
+        <source>Sequential (1)</source>
+        <translation>Sekvenční (1)</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="85"/>
+        <source>Sequential (2)</source>
+        <translation>Sekvenční (2)</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="151"/>
+        <source>Random (1)</source>
+        <translation>Náhodné (1)</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="167"/>
+        <source>Random (2)</source>
+        <translation>Náhodné (2)</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="213"/>
+        <source>Block Size</source>
+        <translation>Velikost bloku</translation>
+    </message>
+    <message>
+        <location filename="../../src/settings.ui" line="229"/>
+        <source>Queues</source>
+        <translation>Front</translation>
```

Comment:
Maybe it is better to use nominativ in plural, front**y**?

Reply:
It is about inflexing - supposing there sentence "How many queues?" (in Czech "Kolik front?")